robolectric-annotations/build.gradle

```diff
@@ -0,0 +1,5 @@
+dependencies {
+    // Compile dependencies
+    provided "com.intellij:annotations:12.0"
+    provided "org.robolectric:android-all:6.0.0_r1-robolectric-0"
```

Comment:
I guess we want to change this to be the android.jar from the Android SDK.

Can do this in a follow up though since this is a like for like change